printer_mods/BladeScraper-Designs/Microswitch_Pod_JST/readme.md

```diff
@@ -0,0 +1,8 @@
+This is a simple modification to the microswitch pod to allow the installation of a JST-XH 4-pin connector.
+
+JST-XH  4-pin connector part number B4B-XH-A(LF)(SN) slides into the hole and there's a small lip to keep it sliding all the way thru.  
+Depending on your tolerances it may be tight enough as is, or you may have to use a bit of CA glue.  You may need to clean up the inside of the slot a bit using a utility knife to get the JST in, as it does involve bridging.
+
+I also slightly increased the size of the screw holes from 1.5mm to 1.7mm as I was having a hard time getting mine screwed down.  
+
+![alt text](https://github.com/BladeScraper-Designs/VoronUsers/blob/master/printer_mods/BladeScraper-Designs/Microswitch_Pod_JST/img/assembly.png?raw=true)
```

Comment:
Please update the URL to be relative.  Something like

    ![Microswitch Pod with Connector](img/assembly.png)

Reply:
Done, I think lol.  Not great at using GH.  Lmk if I did it right.